Userland/Libraries/LibJS/Runtime/VM.cpp

```diff
@@ -823,6 +823,29 @@ ThrowCompletionOr<void> VM::link_and_eval_module(Module& module)
     return {};
 }
 
+static String resolve_filepath(StringView filepath, StringView module_type)
```

Comment:
maybe resolve_module_filepath? This is a pretty generic name

Reply:
\*ahem\*, `filepath` isn't a word, can we have `file_path` pls?

Reply:
ACK